dts/bindings/rtc/st,stm32-rtc.yaml

```
@@ -14,6 +14,24 @@ properties:
   reg:
     required: true
 
+  sync-prescaler:
+    type: int
+    description: |
+      RTC synchronous prescaler value. Optional, defaulting to set values
+      depending on source clock.
```

```suggestion
      RTC synchronous prescaler value. Optional, if not set defaulted in code to:
      - 0x7F if RTC source clock is LSE/LSI
      - 0x7C if RTC source clock is HSE
```

Apply to others as well.
I know some would argue that default should be put in device tree, but conditional defaults is not possible. It would work for last one, but done that way for consistency.

---

Still missing in the branch you pushed.